build-system/tasks/e2e/puppeteer-controller.js

```diff
@@ -16,14 +16,9 @@
 
 const fs = require('fs');
 const puppeteer = require('puppeteer'); // eslint-disable-line no-unused-vars
```

Comment:
As a next step let's consider removing this controller as well, since it is not used.

Reply:
It is used for typing though. You mentioned the potential of getting rid of `puppeteer-controller` in your other comment https://github.com/ampproject/amphtml/pull/33844#discussion_r614222362 so rather than dropping type enforcement or trying to do this all with relative imports I'd rather leave this as is for now and try to remove the whole thing at once.

Reply:
Works for me